onnxscript/ir/passes/common/clear_metadata_and_docstring.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Clear all metadata and docstring from the model, graphs, nodes, and functions."""
+
+from __future__ import annotations
+
+__all__ = [
+    "ClearMetadataAndDocStringPass",
+]
+
+import logging
+
+from onnxscript import ir
+
+logger = logging.getLogger(__name__)
+
+
+class ClearMetadataAndDocStringPass(ir.passes.InPlacePass):
+    def __init__(self) -> None:
+        super().__init__()
+        self.modified = False
+
+    def call(self, model: ir.Model) -> ir.passes.PassResult:
+        # 0. TODO: Should we clean model metadata and docstring?
+
+        # 1. Clean up the graph and the belonged nodes metadata properties
+        self._clear_graph_or_function_metadata_and_docstring(model.graph)
```

Comment:
I would make `_clear_graph_or_function_metadata_and_docstring` return a boolean (modified). This way we don't need to manage a state in the class. Note that currently I think you would need to reset `modified` otherwise.

Reply:
Done